packages/extension-base/src/koni/background/handlers/Extension.ts

```diff
@@ -3108,7 +3108,7 @@ export default class KoniExtension {
         const isChainActive = this.#koniState.getChainStateByKey(chainInfo.slug).active;
 
         if (!isChainActive) {
-          reject(new Error('Please active chain {{chain}} before sign'.replaceAll('{{chain}}', chainInfo.name)));
+          reject(new Error('Please activate chain {{chain}} before signing'.replaceAll('{{chain}}', chainInfo.name)));
```

Comment:
All `chain` words in the extension will be updated to `network`.
Please update this sentence to `Please activate {{chain}} network before signing`

Reply:
Hi @ymittal, thank for your contribution,

Please help me update this sentence.

Reply:
Many thanks,

Have a nice day.